.gitignore

```diff
@@ -76,6 +76,9 @@ web_modules/
 yarn.lock
 .yarn
 
+# npm lockfile
+package-lock.json
+
```

Comment:
This was the only other thing - the sandboxes don't need lockfiles. They should always use the most recent version of the packages specified in the package.json